### PR TITLE
Add favorite game page

### PR DESCRIPTION
**Pull request by yuulianaa**

### CODE REVIEW

File: Net12/WebMaze/Controllers/HomeController.cs [hunk 1]
```diff
@@ -40,6 +40,49 @@ public IActionResult Index()
...
+            var GamesViewModels = new List<GameViewModel>();
+            foreach (var dbGame in _webContext.FavGames)
+            {
+                var gameViewModel = new GameViewModel();
+                gameViewModel.Name = dbGame.Name;
+                gameViewModel.Genre = dbGame.Genre;
+                gameViewModel.YearOfProd = dbGame.YearOfProd;
+                gameViewModel.Desc = dbGame.Desc;
+                gameViewModel.Rating = dbGame.Rating;
+                gameViewModel.Username = dbGame.Username;
```
Comment (TheEclipsE9) [inline Net12/WebMaze/Controllers/HomeController.cs:54]: use dbGame.Creater.Name;

File: Net12/WebMaze/Controllers/HomeController.cs [hunk 2]
```diff
@@ -40,6 +40,49 @@ public IActionResult Index()
...
+        public IActionResult AddGame(GameViewModel gameViewMode)
+        {
+            var dbGame = new Game()
+            {
+                Name = gameViewMode.Name,
+                Genre = gameViewMode.Genre,
+                YearOfProd = gameViewMode.YearOfProd,
+                Desc = gameViewMode.Desc,
+                Rating = gameViewMode.Rating,
+                Username = gameViewMode.Username,
```
Comment (TheEclipsE9) [inline Net12/WebMaze/Controllers/HomeController.cs:77]: Use Creator

File: Net12/WebMaze/EfStuff/DbModel/Game.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+{
+    public class Game
+    {
+        public long Id { get; set; }
+        public string Name { get; set; }
+        public string Genre { get; set; }
+        public int YearOfProd { get; set; }
+        public string Desc { get; set; }
+        public int Rating { get; set; }
+        public string Username { get; set; }
```
Comment (TheEclipsE9) [inline Net12/WebMaze/EfStuff/DbModel/Game.cs:16]: change to public virtual User Creater { get; set; }

**Review by Anastasia1Minsk — COMMENTED** [review]
inherit Game from BaseModel after update

**Review by Anastasia1Minsk — APPROVED** [review]

**Pull request merged**